sdk/highlight-next/src/next-client.tsx

```diff
@@ -29,3 +32,16 @@ export function HighlightInit({
 
 	return null
 }
+
+export type HighlightErrorProps = { errorMessage: string } & ErrorProps
```

Comment:
Export `getHighlightErrorInitialProps` to streamline `pages/_error.tsx` creation.